Cloud Enterprise/Getting Started Examples/aws/terraform/terraform.tfvars.example

```diff
@@ -1,7 +1,12 @@
-# See variables.tf for descriptions
+## See variables.tf for descriptions
 
-profile = "YOUR-AWS-PROFILE"
+project_name = "magical-ece-ansible-and-terraform-mystery-tour"
```

Comment:
haha I like it, but let's keep it shorter, just so its easier for people to see in the AWS console if necessary

Reply:
:) yeah, guess I was figuring people would put their own value in there, but agree the default should probably be sensible